cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -12,6 +12,81 @@ WORKER_NODE_ROOT="${CLUSTER_NAME}-worker"
 
 OPERATOR_GIT_HASH=8d3c30de8ec5a9a0c9eeb84ea0aa16ba2395cd68  # release-4.4
 
+function ensure_pod {
+  namespace=$1
+  label=$2
+
+  intervals=6
+  timeout=10
+
+  if [[ $namespace != "" ]];then
+    namespace="-n $namespace"
+  fi
+
+  if [[ $label != "" ]];then
+    label="-l $label"
+  fi
+
+  count=0
+  until _kubectl get pod $namespace $label; do
+      ((count++)) && ((count == intervals)) && echo "Pods with $label not found" && exit 1
+      echo "[$count/$intervals] Waiting for pods with $label to create.."
+      sleep $timeout
+  done
+}
+
+function ensure_k8s_object {
+  object_type=$1
+  name=$2
+  namespace=$3
+
+  intervals=30
```

Comment:
the retry is happening on more than one function
maybe worth to move the retry logic outside as 
https://gist.github.com/dublx/e99ea94858c07d2ca6de

Reply:
I think this should be done in another PR because there are more code we can export to a helper functions file

Reply:
Its a new code that have this issue couple time, but whatever you want.

Reply:
I understand, I checked the example you sent and in order to implement "retry" function 
It is necessary to pass bash command string to that function, 
I find it a bit risky and prune to bugs and malicious code (even-though this script used by cluster-up), this is why I did not do it in the first place.
What you think?

Reply:
(we talked on other media)

Reply:
ack